src/comm/TcpAcceptor.cc

```diff
@@ -388,6 +388,7 @@ Comm::TcpAcceptor::oldAccept(Comm::ConnectionPointer &details)
     // so we end up with a uniform "(HTTP|FTP-data|HTTPS|...) remote-ip:remote-port"
     fd_open(sock, FD_SOCKET, "HTTP Request");
     details->fd = sock;
+    details->enterOrphanage(); // nobody owns this connection yet
```

Comment:
> this method TcpAcceptor "owns"

I don't think that TcpAcceptor owns this connection - for example, it does not close it explicitly if something goes wrong.

Reply:
Yes, Eduard is right: The comment is correct -- TcpAcceptor does not own `details` (it is just a local variable) -- but I removed the comment in 3aa0783 to avoid wasting time on arguments about the proper interpretation of the word "own" in this context.